src/compiler/crystal/semantic/type_declaration_processor.cr

```diff
@@ -630,11 +630,11 @@ struct Crystal::TypeDeclarationProcessor
   end
 
   private def raise_not_initialized_in_all_initialize(node : ASTNode, name, owner)
-    node.raise "instance variable '#{name}' of #{owner} was not initialized in all of the 'initialize' methods, rendering it nilable"
+      node.raise "instance variable '#{name}' of #{owner} was not initialized directly in all of the 'initialize' methods, rendering it nilable. Indirect initialization is not supported."
```

Comment:
Perhaps the error message can be explained in a more verbose way, similar to `raise_doesnt_explicitly_initializes` multi-line error message?